beacon-chain/rpc/eth/validator/validator.go

```diff
@@ -404,66 +404,65 @@ func (vs *Server) ProduceBlockV2SSZ(ctx context.Context, req *ethpbv1.ProduceBlo
 // ProduceBlindedBlock requests the beacon node to produce a valid unsigned blinded beacon block,
 // which can then be signed by a proposer and submitted.
 //
-// Pre-Bellatrix, this endpoint will return a regular block.
+// Under the following conditions, this endpoint will return an error.
+// - The current time is before bellatrix fork epoch.
+// - The requested slot is before bellatrix fork epoch.
```

Comment:
This is against the spec: https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/publishBlindedBlock
> Pre-Bellatrix, this endpoint will accept a SignedBeaconBlock.

Reply:
That's publish, not produce. but i see it's the same with produce, that's annoying, I'll have to add the original code back: 
https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Validator/produceBlindedBlock

Reply:
It makes no sense to produce a non-blind block IMO